<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClearCacheDialog</class>
 <widget class="QDialog" name="ClearCacheDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelCacheSize">
     <property name="text">
      <string>The cache takes up: ...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonClear1">
     <property name="text">
      <string>Delete all pages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonClear2">
     <property name="text">
      <string>+ covers and infos</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonClear3">
     <property name="text">
      <string>+ progress and favorites</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
